tools/notebooks/boosted-farm.ipynb

```diff
@@ -14,9 +14,197 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[93m2024-01-08 14:46:30,169 - WARNING - Structure definition for stakings_boosted not found in /home/ovi/projects/dex/mx-sdk-py-exchange/deploy/configs-testnet/deploy_structure.json\u001b[0m\n",
+      "\u001b[93m2024-01-08 14:46:30,170 - WARNING - Structure definition for metastakings_boosted not found in /home/ovi/projects/dex/mx-sdk-py-exchange/deploy/configs-testnet/deploy_structure.json\u001b[0m\n",
+      "\u001b[93m2024-01-08 14:46:30,170 - WARNING - Structure definition for position_creator not found in /home/ovi/projects/dex/mx-sdk-py-exchange/deploy/configs-testnet/deploy_structure.json\u001b[0m\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "loaded 10 accounts from 1 PEM files.\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[38;20m2024-01-08 14:46:30,691 - DEBUG - Account.sync_nonce() done: 324\u001b[0m\n",
```

Comment:
clear the outputs before committing any changes for notebooks

Reply:
good catch. I usually do but this one escaped somehow.